core/src/main/kotlin/plugins/OpenApi.kt

```diff
@@ -113,29 +116,53 @@ fun Application.configureOpenApi() {
             generator = { type ->
                 type
                     .collectSubTypes()
-                    .processReflection {
+                    .analyzeTypeUsingReflection {
```

Comment:
BTW, I had started doing the same in 04b71e2628d3f6c076c1a126b2b454ab4859c1bd, but never got around finishing it.

IIRC the `type` in line 117 had to be changed to `initial(type)`.

Reply:
I couldn't find that function, I checked the default generators and they don't call it.